src/components/AccountCard.js

```diff
@@ -25,27 +25,34 @@ import AccountIcon from './AccountIcon';
 import TouchableItem from './TouchableItem';
 
 export default class AccountCard extends React.PureComponent<{
-  title: ?string,
   address: string,
   chainId: string,
-  onPress: () => any
+  onPress: () => any,
+  title: ?string,
```

Comment:
`title?:`

Reply:
I was just moving the props around for alphabetical order, I wonder if this is understood correctly. Will change it.